ext/openssl/openssl.c

```diff
@@ -537,14 +534,14 @@ static time_t php_openssl_asn1_time_to_time_t(ASN1_UTCTIME * timestr) /* {{{ */
 	size_t timestr_len;
 
 	if (ASN1_STRING_type(timestr) != V_ASN1_UTCTIME && ASN1_STRING_type(timestr) != V_ASN1_GENERALIZEDTIME) {
-		php_error_docref(NULL, E_WARNING, "Illegal ASN1 data type for timestamp");
+		zend_value_error("Illegal ASN1 data type for timestamp");
```

Comment:
Please do not change this one as it would be inconsistent with some warnings below. This is used just by `openssl_x509_parse` so there is still value to not fail the whole operation to see the rest of the fields for in this case incorrectly formatted cert.